src/TwigRendererFactory.php

```diff
@@ -48,6 +47,7 @@ public function __invoke(ContainerInterface $container) : TwigRenderer
      * array having precedence.
      *
      * @param array|ArrayObject $config
+     *
```

Comment:
Revert; we do not add whitespace between `@param`, `@return`, or `@throws` annotations.